### ISSUE TITLE
CRAM Dots emulation

### ISSUE THREAD

**Issue by getCursorsExe** [irixxxx/picodrive#150]
Wouldn't be nice to add CRAM dots emulation to make it closer to hardware?

**Comment by dxgldotorg** [comment]
Likely this requires making it cycle accurate.

**Comment by irixxxx** [comment]
No, cycle accuracy is probably not necessary. However, some aspects of the cycle timing in PicoDrive don't allow for this to be properly done, first of all the rendering which is, for speed reasons, actually done on a line base and not on a vdp slot base.
I have been pondering this on and off, and while I have some ideas how this could be achieved there are still some open questions.

**Comment by irixxxx** [comment]
A lot of things in PicoDrive are actually cleverly approximated to reduce computational costs. It is the reason why PicoDrive can still be executed on rather low powered devices while achieving a good emulation quality.
